proto/contract.proto

```diff
@@ -344,6 +355,9 @@ message Resource {
   // - setting the `source` attribute of a KafkaSource event (when it's not a CloudEvent)
   // - tagging metrics
   Reference reference = 11;
+
+  // Name of the service account to use for OIDC authentication.
+  string OIDCServiceAccountName = 12;
```

Comment:
here I am not sure if the `Reference` type would be better or not. That way, we would include the namespace of the SA too, but we can get this information from resource.reference.namespace too :shrug:

Reply:
Shouldn't this be part of "Egress" ?

Reply:
Uh yes :flushed: . Will move it. Thanks for catching.